plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1531,6 +1532,8 @@ modelement.villagerprofession.description=Villager profession defines a version
 modelement.villagertrade.description=Villager trade adds new trades to villager professions.
 preferences.section.ui=User interface
 preferences.section.ui.description=On this page you can change user interface settings and parameters. You will need to restart MCreator if you change these settings in order for them to take effect.
+preferences.section.system=System
```

Comment:
This section should be called "Workspace" or "Behaviour" as system sounds like something for computer or OS settings, not app. If IDEA calls it this way this does not mean it is good naming by default

Reply:
I really had no idea about this section. I had the idea to call it "Workspace", but I thought it could confuse users as it doesn't affect workspaces directly (like workspace settings do). Thanks for the idea though!